zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/Helium.java

```diff
@@ -411,7 +412,7 @@ public HeliumPackageSuggestion suggestApp(Paragraph paragraph) {
    * @return ordered list of enabled buildBundle package
    */
   public List<HeliumPackage> getBundlePackagesToBundle() {
-    Map<String, List<HeliumPackageSearchResult>> allPackages = getAllPackageInfoWithoutRefresh();
+    Map<String, List<HeliumPackageSearchResult>> allPackages = getAllPackageInfo();
```

Comment:
Then, what about rollback all changes and create new API for `/refresh` like things?

@necosta